docs/commands/container.md

```diff
@@ -307,6 +308,33 @@ scw container cron update <cron-id ...> [arg=value ...]
 
 
 
+## Deploy a container
+
+Automatically build and deploy a container.
+
+Automatically build and deploy a container.
```

Comment:
Duplication of content here.

Reply:
Not to be fixed in this PR
Issue: https://github.com/scaleway/scaleway-cli/issues/2837